style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+*:focus {
+  outline: none;
+}
+
+body {
+  position: relative;
+  background: #f0f0f0;
+  font-family: 'roboto', sans-serif;
+}
+
+.navbar {
+  position: sticky;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #3e3c3c;
+  height: 50px;
+  padding: 0 30px 0 30px;
+}
+
+.navbar h4 {
+  font-family: 'inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.toggle-btn {
+  width: 20px;
+  height: 14px;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  align-content: center;
+  justify-content: space-around;
+}
+
+.toggle-btn span {
+  width: 100%;
+  height: 2px;
+  background-color: #fff;
+}
+
+.nav-links {
+  display: none;
+}
+
+.nav-links p {
+  display: inline-block;
+  visibility: hidden;
+  margin-left: 20px;
+  color: #fff;
+  font-family: 'inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+/*  ----headline----  */
+
+.headline {
+  background: #1c1a19;
+  min-height: 100vh;
+  height: 100%;
+  padding: 0 24px 0 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.headline h1 {
+  font-family: 'crete round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.headline h2 {
+  font-family: 'roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  color: #fff5e1;
+}
+
+.headline p {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
```

Comment:
As you are going to have a section for all this content with your changes, now you will be able to fix the `height` view on your screen, since it covers the entry page. As this is only one section of your website it should have a max height.

- [ ] Kindly fix the size of the background so it covers the whole content and doesn't leave any white spaces on the view. 💯 

![image](https://user-images.githubusercontent.com/93630700/198168997-11510514-b82d-4140-a486-5975e2910207.png)

- [ ] Verify the different padding and space between this elements so it's come the most similar to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

![image](https://user-images.githubusercontent.com/93630700/198169279-8e9a3feb-8278-4559-b399-7eefd7da8a8a.png)

Reply:
Done, do I have to comment on all the others as well?

Reply:
@Oscar-Espinoza good work! 💯 . It's not necessary to do the comments unless you have some questions or you want to leave some gratitude to the reviewer. Keep coding!! 🥇